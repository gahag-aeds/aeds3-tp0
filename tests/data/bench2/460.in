460 230
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 16 36 2 52 98 62 94 33 39 25 51 89 20 22 46 62 90 51 33 92 0 1 44 83 89 98 42 63 38 99 
Sub 111 263
Min 140 292
Sub 101 253
Max 148 300
Sub 216 368
Sum 104 256
Sub 134 286
Max 201 353
Sub 144 296
Max 124 276
Sub 30 182
Sum 196 348
Sub 250 402
Max 27 179
Sub 141 293
Sum 75 227
Sub 181 333
Sum 168 320
Add 161 313
Sum 21 173
Add 56 208
Min 60 212
Add 25 177
Max 83 235
Add 103 255
Sum 73 225
Sub 7 159
Max 97 249
Add 203 355
Max 42 194
Add 257 409
Min 6 158
Sub 181 333
Sum 129 281
Add 153 305
Min 168 320
Sub 232 384
Min 58 210
Sub 140 292
Max 69 221
Sub 260 412
Max 169 321
Add 36 188
Max 203 355
Add 196 348
Min 176 328
Add 277 429
Max 148 300
Sub 115 267
Max 210 362
Add 248 400
Sum 250 402
Add 41 193
Sum 272 424
Sub 104 256
Max 143 295
Sub 116 268
Max 110 262
Sub 11 163
Max 11 163
Add 151 303
Max 39 191
Add 16 168
Max 103 255
Sub 168 320
Min 247 399
Sub 270 422
Sum 92 244
Sub 304 456
Max 103 255
Sub 239 391
Sum 256 408
Sub 271 423
Sum 274 426
Sub 230 382
Min 238 390
Sub 222 374
Min 279 431
Sub 158 310
Sum 108 260
Sub 181 333
Sum 98 250
Add 129 281
Max 300 452
Sub 141 293
Min 232 384
Sub 214 366
Max 256 408
Add 35 187
Sum 304 456
Sub 38 190
Sum 264 416
Add 35 187
Max 53 205
Add 60 212
Max 202 354
Sub 71 223
Sum 22 174
Add 294 446
Sum 67 219
Add 203 355
Sum 225 377
Add 94 246
Sum 246 398
Sub 271 423
Max 17 169
Sub 255 407
Sum 264 416
Add 295 447
Min 105 257
Add 64 216
Min 247 399
Sub 281 433
Max 193 345
Add 11 163
Sum 114 266
Add 196 348
Min 303 455
Add 141 293
Max 230 382
Add 299 451
Max 178 330
Sub 266 418
Min 27 179
Add 2 154
Min 18 170
Add 228 380
Sum 72 224
Add 118 270
Min 293 445
Sub 217 369
Sum 14 166
Add 290 442
Sum 4 156
Sub 31 183
Max 253 405
Add 99 251
Sum 285 437
Sub 209 361
Sum 86 238
Sub 83 235
Sum 44 196
Sub 104 256
Max 135 287
Add 8 160
Min 193 345
Sub 249 401
Max 204 356
Add 188 340
Sum 82 234
Sub 205 357
Min 180 332
Sub 260 412
Sum 163 315
Add 206 358
Max 261 413
Sub 180 332
Sum 205 357
Sub 105 257
Max 70 222
Sub 270 422
Max 14 166
Sub 162 314
Sum 252 404
Add 17 169
Min 192 344
Add 276 428
Max 115 267
Add 169 321
Sum 256 408
Add 279 431
Min 23 175
Add 299 451
Sum 154 306
Add 158 310
Min 307 459
Add 39 191
Max 178 330
Add 203 355
Min 227 379
Add 259 411
Max 100 252
Add 178 330
Sum 290 442
Add 35 187
Max 76 228
Sub 16 168
Sum 293 445
Add 52 204
Max 120 272
Sub 55 207
Sum 14 166
Add 133 285
Max 216 368
Add 37 189
Max 229 381
Add 100 252
Min 5 157
Sub 36 188
Sum 41 193
Sub 260 412
Sum 87 239
Add 142 294
Sum 24 176
Sub 240 392
Min 71 223
Sub 23 175
Sum 51 203
Sub 248 400
Max 304 456
Sub 277 429
Min 279 431
Add 175 327
Max 226 378
Sub 33 185
Min 288 440
Sub 73 225
Sum 48 200
Add 274 426
Max 124 276
Add 167 319
Min 21 173
Sub 217 369
Max 72 224
Add 237 389
Max 253 405
Add 300 452
Max 108 260
Add 30 182
Sum 107 259
