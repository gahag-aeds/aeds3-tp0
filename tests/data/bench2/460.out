-1
97
7491
98
95
7228
98
7058
7107
6950
-5
96
7406
97
99
-3
7326
-4
-4
99
92
95
-6
98
97
7574
7568
97
96
100
100
98
-7
7138
97
7069
6943
-9
-8
6930
6870
97
-12
96
6968
6515
102
90
7422
7280
6274
6571
102
6440
-6
-13
90
7223
-8
91
94
-1
1
7765
-7
7932
7929
96
6989
7748
7975
99
-14
90
7712
-15
6870
94
6111
105
105
6033
-18
98
6189
1
6670
-8
94
-14
102
7291
108
7358
101
8068
94
94
2
8305
8029
8305
0
8299
102
-13
92
-9
8073
100
2
104
95
101
7948
